### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Momentum site launch
- Remove cron job
- Submit sitemap for production branch
- Remove basic auth for production
- Lift robot noindex nofollow
- #214 
- #215 

# Review By (Date)
- Jan 31

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code
2. Look at preview

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-280

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -42,10 +42,14 @@
...
   to = "/"
   status = 301
-  force = true
\ No newline at end of file
+  force = true
+
+# CONTEXTUAL PLUGINS
+# ##############################################################################
+[[context.production.plugins]]
+  package = "netlify-plugin-submit-sitemap"
```
Comment (yvonnetangsu) [inline netlify.toml:55]: @sherakama Am I doing this right?

**Review by sherakama — APPROVED** [review]
GTG

**Comment by yvonnetangsu** [comment]
Thanks @sherakama! Let's merge this afternoon after we start our launch Zoom 😄

**Comment by yvonnetangsu** [comment]
@sherakama Want me to add your readme update to this release?

**Comment by sherakama** [comment]
> @sherakama Want me to add your readme update to this release?

Nah

**Pull request merged**